app/controllers/games_controller.rb

```diff
@@ -16,13 +16,22 @@ def create
   end
 
   def show
-    @game = Game.find_by_id(params[:id])
```

Comment:
We still need this line because `show.html.erb` needs to know which game to display, whether or not someone is joining. Or you could just call `#current_game` here since it's defined below.

Reply:
changed back